.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Upload Python Package
+
+on:
+  release:
+    types: [published]
+  push:
+    branches: [ main ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
```

Comment:
Would it possible to have also a trigger for the release candidate tags, but then it would be pushed to a pypi devel package? Would this be useful?

Reply:
@igaw sure, it is controlled by 
```
      if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```
below. once merged and working, we can adjust as we want, no problem

Reply:
Ah I see, so we can do string ops on 'github.ref' to figure out what type of tag it is.

Reply:
exactly